superset/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  superset:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: pudl-superset
+    environment:
+      AUTH0_CLIENT_ID: ${AUTH0_CLIENT_ID}
+      AUTH0_CLIENT_SECRET: ${AUTH0_CLIENT_SECRET}
+      AUTH0_DOMAIN: ${AUTH0_DOMAIN}
+      SUPERSET_SECRET_KEY: ${SUPERSET_SECRET_KEY}
+      SUPERSET_DB_HOST: ${SUPERSET_DB_HOST}
+      SUPERSET_DB_PORT: ${SUPERSET_DB_PORT}
+      SUPERSET_DB_USER: ${SUPERSET_DB_USER}
+      SUPERSET_DB_PASS: ${SUPERSET_DB_PASS}
+      SUPERSET_DB_NAME: ${SUPERSET_DB_NAME}
+    ports:
+      - "8080:8088"
+    volumes:
+      # - {path to your pudl.duckdb file}:/app/pudl.duckdb
+      - ./roles.json:/app/roles.json
```

Comment:
Do we want to try to track the role definitions in git so our local deployments can use the roles we're using in production?